### PR TITLE
1.10.1 release

### PR DESCRIPTION
**Pull request by dasormeter**
Take the 1.10.1 release focused on

`[Daml Driver for PostgreSQL] Enable migrate only option at startup via new --sql-start-mode parameter`

### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [X] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [X] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [X] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [X] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ba6ba9019f0b3496fad68d936989fd9895cf3440 1.11.0-snapshot.20210217.6338.0.ba6ba901
+cc880e290b2311d0bf05d58c7d75c50784c0131c 1.10.1
```
Comment (cocreature) [inline LATEST:2]: that doesn’t look right, this points to https://github.com/digital-asset/daml/commit/cc880e290b2311d0bf05d58c7d75c50784c0131c

Comment (dasormeter) [reply LATEST]: corrected

**Review by cocreature — APPROVED** [review]
:shipit:

File: LATEST [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ba6ba9019f0b3496fad68d936989fd9895cf3440 1.11.0-snapshot.20210217.6338.0.ba6ba901
+9d4b23229ae0f7ee51b424d8c90b42e8bd513446 1.10.1
```
Comment (cocreature) [inline LATEST:2]:
```suggestion
52c8caa76e6abc0e0f17a51562d11b7e97c3ab95 1.10.1
```
bumping to include https://github.com/digital-asset/daml/pull/8903#event-4353521209

**Comment by dasormeter** [comment]
Manual tests passed on [Linux/macOS]

**Comment by cocreature** [comment]
manual tests passed on Windows

**Pull request merged**